app-web/typography.js

```diff
@@ -1,10 +1,11 @@
 import Typography from 'typography';
+import '@bcgov/bc-sans/css/BCSans.css';
 
 const typography = new Typography({
   baseFontSize: '18px',
   baseLineHeight: 1.25,
-  headerFontFamily: ['Noto Sans', 'Verdana', 'Arial', 'sans-serif'],
-  bodyFontFamily: ['Noto Sans', 'Verdana', 'Arial', 'sans-serif'],
+  headerFontFamily: ['BCSans', 'Noto Sans', 'Verdana', 'Arial', 'sans-serif'],
```

Comment:
let'ss remove Noto Sans all together:) We are going to avoid using google fonts that are served over a cdn. Happy to explain why offline!

Reply:
In addition to this you can also remove the reference for google fonts in the `gatsby-config.js` file